js/payment-methods/litecoin.js

```diff
@@ -37,6 +37,37 @@ app.paymentMethods.litecoin = (function() {
 			}
 		],
 
+		networks: {
+			// https://github.com/litecoin-project/litecore-lib/blob/9c3b2712de14335d2a953a8772aee87e23be6cf6/lib/networks.js#L130-L145
+			mainnet: _.extend({}, bitcoin.networks.litecoin, {
+				// https://github.com/litecoin-project/litecoin/blob/ba8ed3a93be7e7a97db6bc00dd7280fa2f1548bc/src/chainparams.cpp#L137-L138
+				bip32: {
+					public: (new Buffer([0x04, 0x88, 0xB2, 0x1E])).readUInt32BE(0),
+					private: (new Buffer([0x04, 0x88, 0xAD, 0xE4])).readUInt32BE(0)
+				},
+				// public key:
+				pubKeyHash: 0x30,
+				// script:
+				scriptHash: 0x32,
+				// private key:
+				wif: 0xB0
+			}),
+			// https://github.com/litecoin-project/litecore-lib/blob/9c3b2712de14335d2a953a8772aee87e23be6cf6/lib/networks.js#L156-L162
+			testnet: _.extend({}, bitcoin.networks.litecoin, {
```

Comment:
@chill117  Sooooo...could you tell me real quick where did you find the correct values? :)

Reply:
From the [litecore-lib](https://github.com/litecoin-project/litecore-lib/) project on github. It's forked from the bitcore-lib project (made by BitPay).